hubble-proxy/cmd/completion/completion.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
nit: I think we could just use SPDX here to trim down the license boilerplate for these new files?

Reply:
We should discuss this broader in #development on Slack and then apply it universally.

Reply:
Indeed, I think this should be tackled as part of #9846.